.editorconfig

```diff
@@ -0,0 +1,12 @@
+# editorconfig.org
```

Comment:
.editorconfig files still have no effect on my eclipse, if you know anybody with eclipse that has it working send them my way.